Tools/LyTestTools/ly_test_tools/o3de/asset_processor.py

```diff
@@ -111,7 +111,7 @@ def wait_for_idle(self, timeout=DEFAULT_TIMEOUT_SECONDS):
 
         self.send_message("waitforidle")
         result = self.read_message(read_timeout=timeout)
-        assert self.process_exists(), "Asset Processor appears unexpectedly shut down, or has crashed"
+        assert self.process_exists(), "Asset Processor has unexpectedly shut down during idle wait!"
```

Comment:
I don't see how this message is clearer. Removing suggestion of a possible crash seems to make it less helpful. Also the exception will already include wait_for_idle in its stacktrace.

minor: exclamation marks in error messages can feel a bit aggressive / annoying when debugging.

Reply:
I don't mind pointing out when the exception occurs to make it more readable, but can add back the suggestion of a crash:
`Asset Processor has crashed or unexpectedly shut down during idle wait.`